interface_tester/collector.py

```diff
@@ -173,6 +175,7 @@ def _gather_charms_for_version(version_dir: Path) -> Optional[_InterfacesDotYaml
 
     providers = charms.get("providers", [])
     requirers = charms.get("requirers", [])
+    owners = charms.get("owners") or []
```

Comment:
if we're doing this `.get(x) or []` we should probably apply it to the two lines above as well, for consistency.